backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -145,14 +144,16 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
   // keeping track of the paths cleanly without so many value mappers 
   def mapCommandLineJobInputWomFile(womFile: WomFile): WomFile = mapCommandLineWomFile(womFile)
 
+  def inputsToNotLocalize: Set[WomFile] = Set.empty
```

Comment:
Would be nice to explain here that this optimization is flipped on/off on a per-backend basis, with a default of off